guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<p>For the multi-text class image, can you update it such that the zeros and ones are randomly spread? It is not clear to me if the class values are binary (0 /1 ) or if they can take numeric values (greater than 1, perhaps corresponding to the number of times each class occurs in the text). May be you can explain this in the text following the image.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>The dataset from where the image is taken is from <a href="https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge" target="_blank">Toxic comment classification challenge</a>, which contains comments with very strong and sometimes abusive words. We have tried not to show such text which taking the image as the content would not be appropriate for our users. May be we can keep the image as it is. I agree with your 2nd point. I will be making necessary changes to the text following the image</p>